prisma/schema.prisma

```diff
@@ -159,25 +159,30 @@ model Image {
 // this is for validator service
 model CandidUserInfo {
   id               Int     @id @default(autoincrement())
+  createdAt       DateTime @default(now())
+  updatedAt       DateTime @updatedAt
+
   claimId          Int?    @unique
   firstName        String?
   lastName         String?
   candid_entity_id String
   email            String
   profileURL       String
+  response         ResponseStatus?
```

Comment:
should response be decoupled, maybe they sent out 20 validations, have 20 responses

responseStatus should be with the request, not the user i think?

with the user maybe we have a score, i feel like it oculd be in a separate table tho?